pyiceberg/avro/codecs/__init__.py

```diff
@@ -26,15 +26,21 @@
 
 from __future__ import annotations
 
-from typing import Dict, Optional, Type
+from typing import Dict, Literal, Optional, Type
+
+from typing_extensions import TypeAlias
 
 from pyiceberg.avro.codecs.bzip2 import BZip2Codec
 from pyiceberg.avro.codecs.codec import Codec
 from pyiceberg.avro.codecs.deflate import DeflateCodec
 from pyiceberg.avro.codecs.snappy_codec import SnappyCodec
 from pyiceberg.avro.codecs.zstandard_codec import ZStandardCodec
 
-KNOWN_CODECS: Dict[str, Optional[Type[Codec]]] = {
+AvroCompressionCodec: TypeAlias = Literal["null", "bzip2", "snappy", "zstandard", "deflate"]
```

Comment:
In other places in the project we use zstd instead of zstandard. What do you think about using zstd?

Reply:
Hey @ndrluis that's a great catch! We should use `zstd` instead. Avro uses `zstandard`, that's why I copied it here, but it is actually wrong.

Reply:
keep in min that this `TypeAlias` is called `AvroCompressionCodec`, so it should actually use `zstandard` instead of `zstd`, but I've fixed it for Iceberg. LMKWYT